### DISCUSSION TITLE
interfacialization.exe [options] [file_or_dir] [file_or_dir] [...] interfacialization.exe: error: unrecognized arguments: --alluredir --clean-alluredir   inifile: None   rootdir: C:\Users\BG301

### DISCUSSION THREAD

**Discussion by lz452678** [pyinstaller/pyinstaller#8113] [Help]
--alluredir --clean-alluredir 这两个命令为啥报错了，有解决方案吗？望告知谢谢

**Comment by rokm** [comment]
Can you provide an example code that triggers this error? 

Based on the limited information you provided, this looks like a component (*allure*?) trying to spawn a worker sub-process using `sys.executable`, and running afoul of command-line parser in your entry-point script. If that is the case, you need to detect this invocation (based on parameters) before you do your main program's CLI parsing, and divert the program flow into whatever is supposed to happen in the allure subprocess...
